### DISCUSSION TITLE
multi gpu on machine without slurm

### DISCUSSION THREAD

**Discussion by reiselmillan** [ACEsuit/mace#323] [Q&A]
Hi, 
Is it possible to train a model on a computer with several GPUs but  without SLURM distributed system? For instance, a local workstation with four GPUs.
Thanks in advance

**Comment by ilyes319** [comment]
I think this PR: https://github.com/ACEsuit/mace/pull/143 should enable you to do that.

**Reply by reiselmillan** [reply]
thanks
